pkg/operator/ceph/controller/spec.go

```diff
@@ -813,6 +813,30 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 	}
 }
 
+// rgw operations will be logged in sidecar rgw-ops-log
+func RgwOpsLogContainer(daemonID, ns string, c cephv1.ClusterSpec, rgwOpsLogResources v1.ResourceRequirements) *v1.Container {
+
+	// Generate ops log file name based on daemon ID
+	opsLogFile := fmt.Sprintf("ops-log-%s.log", daemonID)
+
+	return &v1.Container{
+		Name: "rgw-ops-log",
+		Command: []string{
+			"/bin/bash",
+			"-m", // Terminal job control, allows job to be terminated by SIGTERM
+			"-c", // Command to run
+			fmt.Sprintf("tail -n+1 -F %s", path.Join(config.VarLogCephDir, opsLogFile)),
+		},
+		Image:           c.CephVersion.Image,
+		ImagePullPolicy: GetContainerImagePullPolicy(c.CephVersion.ImagePullPolicy),
+		VolumeMounts:    DaemonVolumeMounts(config.NewDatalessDaemonDataPathMap(ns, c.DataDirHostPath), "", c.DataDirHostPath),
```

Comment:
Some questions:
1. Did you test this is working already? I just can't remember if the `/var/log/ceph` dir is one of the host paths. If so, the main rgw pod must be mounting it as well, so both containers have access to it.
2. If multiple rgw pods are running on the same node, they would be using the same host path, and the sidecars would pick up all the logs from all rgw pods on the node. 
3. IIRC rook is already rotate logs under `/var/log/ceph`, but we will need to confirm if this ops log is rotated as well.

Reply:
added the ops log rotation if logcollector is enabled in the recent comment

Reply:
You can try with this container https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/spec.go#L158

Reply:
@travisn @ideepika 
Rook actually change the `/etc/logrotate.d/ceph` file, as per the daemon container

For example:
```
/var/log/ceph/ceph-mon.a.log {
    rotate 7
    daily
    maxsize 500M
    compress
    sharedscripts
    postrotate
        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x "ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror" || true
    endscript
    missingok
    notifempty
    su root ceph
}

```

So if we are creating a new container that will create a new file, for which we need to create a separate log rotate container to rotate it,
Or
We may only create the ops log file for rgw